spec/serializers/prison_transfer_reason_serializer_spec.rb

```diff
@@ -26,6 +26,6 @@
   end
 
   it 'contains a `disabled_at` attribute' do
-    expect(result[:data][:attributes][:disabled_at]).to eql disabled_at
+    expect(result[:data][:attributes][:disabled_at]).to eql disabled_at.iso8601
```

Comment:
That's because previously this spec wasn't testing a full round trip of the serializer (converting to JSON and then parsing back), but instead inspecting Ruby objects in the `serializable_hash`. I've tried to make all of the serializer specs more complete and representative by always generating JSON output and then parsing.

That does mean in some cases (depending on the individual spec) I've had to tweak expectations slightly to check the correct object type is returned - e.g. here we do indeed want dates in `YYYY-MM-DD` format for the JSON output.